src/event.h

```diff
@@ -29,21 +29,30 @@ enum {
 };
 
 struct event_motion {
-	int type;
+	struct {
+		int type;
+		int dev;
+	};
```

Comment:
Nameless structs are not valid C89. Also why make a struct here instead of just adding a dev field in the existing struct?

Reply:
This is just to be consistent with the `spnav_event` union such that both `type` and `dev` become accessible without using a nested struct.